sale_channel_partner_binding/views/res_partner.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <data>
+        <record id="view_partner_form" model="ir.ui.view">
+            <field name="model">res.partner</field>
+            <field name="inherit_id" ref="base.view_partner_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//page[@name='sales_purchases']//group[@name='sale']" position="inside">
+                    <field name="partner_bind_ids" attrs="{'readonly': 1}">
```

Comment:
we should still give the possibility to create/delete a binding manually so remove  the readonly stuff